src/web/components/LiveBlock.tsx

```diff
@@ -0,0 +1,336 @@
+		<div
+			className={css`
+				& figcaption {
+					margin-left: ${ASIDE_WIDTH}px;
+					padding-left: ${GUTTER}px;
+					padding-right: ${GUTTER}px;
+				}
+			`}
+		>
+			{children}
+		</div>
+	);
+};
+
+const BlockText = ({ children }: { children: React.ReactNode }) => {
+	return (
+		// Set a left margin the same with as the left aside column plus
+		// padding gutters on both sides
+		<div
+			className={css`
+				margin-left: ${ASIDE_WIDTH}px;
+				padding-left: ${GUTTER}px;
+				padding-right: ${GUTTER}px;
+			`}
+		>
+			{children}
+		</div>
+	);
+};
+
+const LastUpdated = ({
+	lastUpdatedDisplay,
+	lastUpdated,
+}: {
+	lastUpdatedDisplay: string;
+	lastUpdated: number;
+}) => {
+	return (
+		<div
+			className={css`
+				display: flex;
+				align-items: flex-end;
+				${textSans.xsmall()};
+				color: ${neutral[46]};
+			`}
+		>
+			<time dateTime={new Date(lastUpdated).toISOString()}>
+				{`Updated at ${lastUpdatedDisplay}`}
+			</time>
+		</div>
+	);
+};
+
+const FirstPublished = ({
+	firstPublished,
+	blockLink,
+}: {
+	firstPublished: number;
+	blockLink: string;
+}) => {
+	const publishedDate = new Date(firstPublished);
+	return (
+		<a
+			href={blockLink}
+			data-ignore-global-link-styling={true}
+			// title={publishedDate.toLocaleString()}
+			className={css`
+				${textSans.xsmall({ fontWeight: 'bold' })}
+				margin-bottom: ${space[1]}px;
+				padding-top: ${space[1]}px;
+				display: flex;
+				flex-direction: row;
+				${from.phablet} {
+					flex-direction: column;
+				}
+				text-decoration: none;
+				:hover {
+					filter: brightness(30%);
+				}
+			`}
+		>
+			<time
+				dateTime={publishedDate.toISOString()}
+				className={css`
+					color: ${neutral[20]};
+				`}
+			>
+				{makeRelativeDate(firstPublished, {
+					format: 'med',
+				})}
+			</time>
+			<span
+				className={css`
+					${textSans.xsmall()};
+					color: ${neutral[46]};
+				`}
+			>
+				{`${publishedDate.getHours()}:${publishedDate.getMinutes()}`}
+			</span>
+		</a>
+	);
+};
+
+const typesWeStretch: CAPIElement['_type'][] = [
+	'model.dotcomrendering.pageElements.YoutubeBlockElement',
+	'model.dotcomrendering.pageElements.ImageBlockElement',
+	'model.dotcomrendering.pageElements.VideoBlockElement',
+	'model.dotcomrendering.pageElements.VideoFacebookBlockElement',
+	'model.dotcomrendering.pageElements.VideoVimeoBlockElement',
+	'model.dotcomrendering.pageElements.WitnessBlockElement',
+	'model.dotcomrendering.pageElements.VineBlockElement',
+	'model.dotcomrendering.pageElements.MultiImageBlockElement',
+	'model.dotcomrendering.pageElements.MediaAtomBlockElement',
+	'model.dotcomrendering.pageElements.GuVideoBlockElement',
+	'model.dotcomrendering.pageElements.EmbedBlockElement',
+	'model.dotcomrendering.pageElements.VideoYoutubeBlockElement',
+];
+
+export const LiveBlock = ({
+	format,
+	block,
+	pageId,
+	webTitle,
+	abTests,
+	adTargeting,
+	host,
+}: Props) => {
+	if (block.elements.length === 0) return null;
```

Comment:
Do we want to return `null` ?

Reply:
What are you thinking in particular? And empty object?...just curious 😄

Reply:
I've learnt to always expect the unexpected at the Guardian, If a thing is possible, it will happen. But I didn't want to have guards all though the code and I'm confident we never want to render an empty block so by returning `null` it is signalling to React to ignore this component (React will pass over `null`s)